boards/arm/qemu_cortex_m3/qemu_cortex_m3.yaml

```diff
@@ -5,5 +5,7 @@ arch: arm
 toolchain:
   - zephyr
   - gccarmemb
+supported:
+  - slip
```

Comment:
What do you mean we support 'networking'?  Can I do ethernet with qemu?  This is meant to try and convey the form of networking we support on qemu targets. (another way to think of this would be what driver(s) would need to be enabled).